modules/object-cache/persistent-object-cache-health-check/load.php

```diff
@@ -0,0 +1,252 @@
+			'label' => 'persistent_object_cache',
+			'test'  => 'perflab_oc_health_persistent_object_cache',
+		);
+	}
+
+	return $tests;
+}
+add_filter( 'site_status_tests', 'perflab_oc_health_add_tests' );
+
+/**
+ * Callback for `persistent_object_cache` health check.
+ *
+ * @since 1.0.0
+ *
+ * @return array The health check result suggesting persistent object caching.
+ */
+function perflab_oc_health_persistent_object_cache() {
+	/**
+	 * Filter the action URL for the persistent object cache health check.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $action_url Learn more link for persistent object cache health check.
+	 */
+	$action_url = apply_filters(
+		'perflab_oc_site_status_persistent_object_cache_url',
+		/* translators: Localized Support reference. */
+		__( 'https://wordpress.org/support/article/optimization/#object-caching', 'performance-lab' )
+	);
+
+	$result = array(
+		'test'        => 'persistent_object_cache',
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => __( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => sprintf(
+			'<p>%s</p>',
+			__( "WordPress performs at its best when a persistent object cache is used. A persistent object cache helps to reduce load on your SQL server significantly and allows WordPress to retrieve your site's content and settings much faster.", 'performance-lab' )
+		),
+		'actions'     => sprintf(
+			'<p><a href="%s" target="_blank" rel="noopener">%s <span class="screen-reader-text">%s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a></p>',
+			esc_url( $action_url ),
+			__( 'Learn more about persistent object caching.', 'performance-lab' ),
+			/* translators: Accessibility text. */
+			__( '(opens in a new tab)', 'performance-lab' )
+		),
+	);
+
+	if ( wp_using_ext_object_cache() ) {
+		$result['label'] = __( 'A persistent object cache is being used', 'performance-lab' );
+
+		return $result;
+	}
+
+	/**
+	 * Filter whether to suggest using a persistent object cache.
+	 *
+	 * Plugin and theme authors should NOT use this filter to discourage the use of an object cache.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param bool $suggest Whether to suggest using a persistent object cache.
+	 */
+	if ( ! apply_filters( 'perflab_oc_site_status_suggest_persistent_object_cache', false ) ) {
+		$result['label'] = __( 'A persistent object cache is not required', 'performance-lab' );
+
+		return $result;
+	}
+
+	$available_services = perflab_oc_health_available_object_cache_services();
+
+	$notes = __( 'Speak to your web host about what persistent object caches are available and how to enable them.', 'performance-lab' );
+
+	if ( ! empty( $available_services ) ) {
+		$notes .= sprintf(
+			/* translators: Available object caching services. */
+			__( ' Your host appears to support the following object caching services: %s.', 'performance-lab' ),
+			implode( ', ', $available_services )
+		);
```

Comment:
Translation strings shouldn't have any trailing spaces, as those get easily missed.
```suggestion
		$notes .= ' ' . sprintf(
			/* translators: Available object caching services. */
			__( 'Your host appears to support the following object caching services: %s.', 'performance-lab' ),
			implode( ', ', $available_services )
		);
```